ui/kotlinx-coroutines-android/resources/META-INF/com.android.tools/r8-from-1.6.0/coroutines.pro

```diff
@@ -5,6 +5,18 @@
     boolean FAST_SERVICE_LOADER_ENABLED return false;
 }
 
--assumenosideeffects class kotlinx.coroutines.internal.FastServiceLoader {
+-assumenosideeffects class kotlinx.coroutines.internal.FastServiceLoaderKt {
     boolean ANDROID_DETECTED return true;
+}
+
+# Disable support for "Missing Main Dispatcher", since we always have Android main dispatcher
+-assumenosideeffects class kotlinx.coroutines.internal.MainDispatchersKt {
+    boolean SUPPORT_MISSING return false;
+}
+
+# Statically turn off all debugging facilities and assertions
+-assumenosideeffects class kotlinx.coroutines.DebugKt {
+    boolean getASSERTIONS_ENABLED() return false;
+    boolean getDEBUG() return false;
+    boolean getRECOVER_STACK_TRACES() return false;
```

Comment:
This fact should be reflected in documentation to android module, stacktracktrace recovery KDoc and related `.md` doc

Reply:
Done